javascript/javascript3/week1/weatherApi.js

```diff
@@ -0,0 +1,46 @@
+const cityName = document.querySelector("input");
+const btn = document.getElementById("btn");
+const result = document.getElementById("weatherstatus");
+
+const empty = document.getElementById("empty");
+btn.addEventListener ('click',() => {
+     
```

Comment:
We should get the cityName value when button is clicked not before that.

Reply:
Hi Piyush .
I didn't understand what do you mean. 
do you mean it should be btn.onclick = () =>{},instead of using addEventListener?

Reply:
I mean we should get inside the listener function